lib/regions.js

```diff
@@ -1,18 +1,23 @@
 const CLEVERTAP_REGIONS = {
-    EUROPE: 'eu1',
-    INDIA: 'in1',
-    SINGAPORE: 'sg1',
-    US: 'us1'
-}
+  EUROPE: "eu1",
+  INDIA: "in1",
+  SINGAPORE: "sg1",
+  US: "us1",
```

Comment:
@KambleSonam we need to add the following regions to this -
Indonesia - aps3
Middle East (UAE) - mec1